compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -564,6 +564,37 @@ fn transcribe_metavar_expr<'a>(
         span
     };
     match *expr {
+        MetaVarExpr::Concat(ref elements) => {
+            let mut concatenated = String::new();
+            for element in elements.into_iter() {
+                let s = 'string: {
+                    if !element.is_var {
+                        break 'string element.ident.to_string();
+                    }
```

Comment:
Wouldn't this be simpler as just `if !element.is_var {} else {}`? or a `match element.is_var { false => , true =>  }`

Reply:
The end result is the same. Regardless, the code was modified according to your comment.